modules/mono/glue/GodotSharp/GodotSharp/Core/SignalAwaiter.cs

```diff
@@ -15,7 +15,7 @@ public SignalAwaiter(Object source, StringName signal, Object target)
         }
 
         [MethodImpl(MethodImplOptions.InternalCall)]
-        internal static extern Error godot_icall_SignalAwaiter_connect(IntPtr source, IntPtr signal, IntPtr target, SignalAwaiter awaiter);
+        internal static extern void godot_icall_SignalAwaiter_connect(IntPtr source, IntPtr signal, IntPtr target, SignalAwaiter awaiter);
```

Comment:
For example, I just realized `SignalAwaiter` should throw an exception if this function returns an error. Rather than silently returning an awaitable that will never complete.